src/features/home/DashboardPanelLeft.tsx

```diff
@@ -22,14 +22,19 @@ interface DashboardPanelLeftProps {
 
 export const DashboardPanelLeft = ({ lbtcUnit, priceFeed }: DashboardPanelLeftProps): JSX.Element => {
   const navigate = useNavigate();
-  const { currency, network } = useTypedSelector(({ settings }: RootState) => settings);
+  const { currency, network, assets } = useTypedSelector(({ settings }: RootState) => settings);
   const { data: listMarkets } = useListMarketsQuery();
   const { data: prices, isLoading: isLoadingPrices, isError: isErrorPrices } = priceFeed;
   const activeMarkets = listMarkets?.marketsList.filter((m) => m.tradable).length || 0;
   const pausedMarkets = (listMarkets?.marketsList.length ?? 0) - activeMarkets;
   //
   const markets = listMarkets?.marketsList.map((m) => m.market as Market.AsObject);
-  const { data: totalCollectedSwapFees } = useTotalCollectedSwapFeesQuery(markets);
+  const { data: totalCollectedSwapFees } = useTotalCollectedSwapFeesQuery({
+    markets: markets,
+    prices: prices,
+    assets: assets,
+    network: network,
+  });
 
   const totalCollectedSwapFeesAsFavoriteCurrency = convertAmountToFavoriteCurrency({
```

Comment:
The `totalCollectedSwapFeesAsFavoriteCurrency` is wrong because `totalCollectedSwapFees` is expressed in sats but interpreted as current `lbtcUnit`. I think the easiest way to solve it is to convert `totalCollectedSwapFees` to `lbtcUnit`. Otherwise, in addition to `preferredLbtcUnit` we need `currentAmountUnit`. But it makes the code harder to understand imo.
By the way you can use tdex app on regtest to make trades and test the code.

Reply:
You are right, I'll make those changes. 

And thanks for the idea on using tdex-app as a way to test this.